app/javascript/components/App.js

```diff
@@ -10,8 +10,23 @@ import LeaderBoard from "./pages/LeaderBoard";
 import MetricsNew from "./pages/MetricsNew";
 import MetricsShow from "./pages/MetricsShow";
 import MetricsEdit from "./pages/MetricsEdit";
+// import MockMetrics from "./MockMetrics"
```

Comment:
This import should not be commented out based on the Trello card.

Reply:
Resolved.